platforms/ide/problems/build.gradle.kts

```diff
@@ -65,6 +65,10 @@ dependencies {
     implementation(projects.logging)
     implementation(projects.messaging)
 
+    testImplementation(projects.stdlibKotlinExtensions)
+
```

Comment:
```suggestion
```

Reply:
I wanted to separate project from library dependencies